### PR TITLE
Fido2

### PR DESCRIPTION
**Pull request by arthurchan35**
From: https://github.com/liferay-frontend/liferay-portal/pull/354

### CODE REVIEW

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/setup_completed.jsp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<div class="sheet-section">
+	<liferay-ui:message key="you-can-register-another-fido2-authenticator-after-removing-currently-registered-one" />
+
+	<aui:input name="mfaRemoveExistingSetup" type="hidden" value="<%= true %>" />
+</div>
+
+<div class="sheet-footer">
+	<button class="btn btn-danger" type="submit">
+		<liferay-ui:message key="button-remove-registered-fido2-authenticator" />
```
Comment (arthurchan35) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/setup_completed.jsp:27]: I added a button to remove registered FIDO2 authenticator.

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/setup.jsp [hunk 2]
```diff
@@ -29,9 +29,11 @@ String pkccOptions = (String)request.getAttribute("pkccOptions");
 				"pkccOptions", pkccOptions
 			).build()
 		%>'
-		label="start"
+		label="register"
 		propsTransformer="js/RegistrationTransformer"
 	/>
+
+	<liferay-ui:message key="button-register-a-fido2-authenticator" />
```
Comment (arthurchan35) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/setup.jsp:36]: Per Zsigmond's request to make button more meaningful.

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/verify_browser.jsp [hunk 3]
```diff
@@ -29,9 +29,11 @@ String assertionRequest = (String)request.getAttribute("assertionRequest");
 				"assertionRequest", assertionRequest
 			).build()
 		%>'
-		label="start"
+		label="verify"
 		propsTransformer="js/AuthenticationTransformer"
 	/>
+
+	<liferay-ui:message key="button-verify-identity-using-a-registered-fido2-authenticator" />
```
Comment (arthurchan35) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/verify_browser.jsp:36]: Same

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by julien** [comment]
Hi Arthur, 

Thanks for re-submitting this.

I took at look at this and just wanted to tell you that you shouldn't hesitate to edit my commit messages because they don't have the correct LPS number and that will also make the `ci:test:sf` fail - you can even squash them as needed if you want to.

Other than that, I'll let the rest of the @liferay-frontend team have a look at this :+1:

**Comment by jbalsas** [comment]
Hey @julien, let's split our code to a different task so @arthurchan35 doesn't need to push it with the rest of their code.

Could you create one task to add `propsTransformer` support to `clay:button` and work out the necessary commits there? That way we can rebase this and get rid of the unnecessary bits.

**Comment by julien** [comment]
@jbalsas seems like a good idea, I'll do that thanks!

**Comment by jbalsas** [comment]
So, the security team informed me they're handling this already, so closing here!! 🙏

**Pull request closed without merging**